docs/output-files/gam-gamma-factors.md

```diff
@@ -2,7 +2,7 @@
 layout: default
 title: (*.gam) Gamma factors
 parent: /output-files.html
-nav_order: 1
+nav_order: 10
 nav_exclude: 1
 ---
```

Comment:
After output file number ten, again, I realize who time-consuming both the writing but also the reviewing is :-) I will ask all co-authors, Danny, Fabio, Susi, and Marc, of the JOVE tutorial to proofread and correct one of the main panels sim (RB), video proc. (Danny), trace processing (Fabio), hist analysis (Susi), transition analysis (Marc).

Any suggestions, how to manage the proofreading in the best way. It must be done via github too, right?

Reply:
Yes Github and Bundle are the best to my opinion.